keyboards/pulsar/xboard_qs/rules.mk

```diff
@@ -0,0 +1,19 @@
+# Build Options (change yes to no to disable)
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug(+400)
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       	# Enable keyboard backlight functionality
+LED_MATRIX_ENABLE = no			#
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow (WS2811, WS2812, WS2812B, WS2812C, SK6812, SK6812MINI, SK6805, APA102)
+RGB_MATRIX_ENABLE = yes			#
+
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes
+
+DYNAMIC_MACRO_ENABLE = yes
+WPM_ENABLE = yes
+
+ENCODER_ENABLE = yes        # Enable Encoder
```

Comment:
```suggestion
# This file intentionally left blank
```

Reply:
Thank you, but,
How can I enable the "Dynamic_MACRO_ENABLE" function?

Reply:
Is there a reason you need this feature enabled?

Reply:
thank you! 
I'll delete it